src/hermes_claude_agent_sdk/__init__.py

```
@@ -1,21 +1,61 @@
+    "RUNTIME_ID",
+    "__version__",
+    "build_runtime_descriptor",
+    "check_compatibility",
+    "create_runtime",
+    "doctor",
+    "doctor_json",
+    "register",
+    "runtime_descriptor",
+    "register_provider_profile",
+    "runtime_factory",
+]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)